apps/institution-service/cypress/e2e/institutions.cy.ts

```diff
@@ -1,48 +1,179 @@
 import { PORT } from "../../src/shared/const";
 import { CachedInstitution } from "../../src/tasks/loadInstitutionsFromJson";
 
+const keysUrl = `http://localhost:8089/v1/clients/keys`;
+
+type Keys = {
+  clientId: string;
+  clientSecret: string;
+};
+
+const getLocalStorage = (args: {
```

Comment:
The whole login process seems more complex than it needs to be. There is a getLocalStorage function, we're setting something in local storage via a command, and the login function is more complex than it needs to be.

We would have a lot less lines of code if we exported a paired down version of the login function and grabbed the access token directly instead of storing and retrieving it from localstorage